schedule_scaling/schedule_scaling.py

```diff
@@ -19,6 +20,8 @@ def create_job_directory():
     if os.path.isdir(temp__dir):
         shutil.rmtree(temp__dir)
     pathlib.Path(temp__dir).mkdir(parents=True, exist_ok=True)
+    # need to copy cadeployment.py to override pykube.Deployment with the correct api version
```

Comment:
why does it need to be copied over to the temp dir? The file is already in the project